exporter/clickhouseexporter/README.md

```diff
@@ -317,283 +333,6 @@ service:
       exporters: [ clickhouse ]
 ```
 
-## Schema
```

Comment:
what is the reason to remove this section?

Reply:
It just bloats up the readme and requires managing it in two places. The scheme definition is in the code.

Reply:
As a rule of thumb, it's always best to keep the PRs to one purpose and make unrelated changes in separate PRs. Otherwise, it's confusing to the reviewers who would ask "Why does improving clickhouse connection management need removing these blocks?"

Reply:
> The scheme definition is in the code.

Where can I find the schema definition in code? I was referencing the definition in the README (for some internal documentation) and I would like to point to the correct location.

Reply:
> > The scheme definition is in the code.
> 
> Where can I find the schema definition in code? I was referencing the definition in the README (for some internal documentation) and I would like to point to the correct location.

For each table there is a `CREATE TABLE` statement like this:

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/4f27accf111db991b87a19f658f375d076390669/exporter/clickhouseexporter/internal/gauge_metrics.go#L31-L64

Reply:
Thanks!